src/compiler/crystal/tools/formatter.cr

```diff
@@ -2058,9 +2061,16 @@ module Crystal
 
       # This is the case of an enum member
       if node.name[0].ascii_uppercase? && @token.type == :","
```

Comment:
Is there any advantage of supporting both the comma and space as member delimiters in enums?

Reply:
That space is supported is a bug. Only comma or newline should be supported. Please open a bug for this.

Reply:
Excluding the comma, why supporting both the comma and new line (and therefore the semicolon)?
```cr
enum E
  A; B; C
  D, E, F
end
```

Reply:
I copied the comma from C#. I wouldn't mind dropping it, but it's a breaking change (I don't mind either)